README.md

```diff
@@ -16,9 +16,7 @@ Quick Start
 *Building Gotcha* is trivial. In the root directory of the repo
 
 ```
-mkdir build
-cd build
-cmake -DCMAKE_INSTALL_PREFIX = <where you want the sofware> ..
+cmake -DCMAKE_INSTALL_PREFIX=<where you want to install GOTCHA> ..
```

Comment:
Hey Tony, I don't think this will work, if you're in the root directory of the repo ".." is above the root. Suppose I'm in `~/src/gotcha` (the repo root). If I `cmake ..` I'm getting `~/src`. That said, I can see how the instructions as written are confusing, any ideas on a wording that makes sense?